python/lsst/daf/butler/script/queryDatasets.py

```diff
@@ -95,7 +95,10 @@ def getAstropyTable(self, datasetTypeName: str) -> AstropyTable:
             raise RuntimeError(f"No DatasetRefs were provided for dataset type {datasetTypeName}")
 
         refInfo = next(iter(self.datasetRefs))
-        dimensions = list(refInfo.datasetRef.dataId.full.keys())
+        dimensions = [
+            refInfo.datasetRef.dataId.universe.dimensions[k]
+            for k in refInfo.datasetRef.dataId.dimensions.data_coordinate_keys
```

Comment:
This seems a bit more convoluted than the original. I'm trying to understand why the `data_coordinate_keys` aren't enough given that they are converted to strings in the next line.